carbon.go

```diff
@@ -384,7 +384,9 @@ func (c *Carbon) String() string {
 // AddYears adds a year to the current time.
 // Positive values travel forward while negative values travel into the past
 func (c *Carbon) AddYears(y int) *Carbon {
-	return NewCarbon(c.AddDate(y, 0, 0))
+	c.Time = c.AddDate(y, 0, 0)
+
+	return c
```

Comment:
if we are doing this why do we need the return?
`
c := Carbon.Now()
a := c. AddYears(2)
`
in this case `a == c`

I agree that we should do it. But we should remove the return.